uamqp/async_ops/client_async.py

```diff
@@ -313,16 +314,18 @@ def __init__(
         :rtype: ~uamqp.message.Message
         """
         while not await self.auth_complete_async():
-            await asyncio.sleep(0.05)
-        response = await asyncio.shield(self._session.mgmt_request_async(
-            message,
-            operation,
-            op_type=op_type,
-            node=node,
-            callback=callback,
-            encoding=self._encoding,
-            debug=self._debug_trace,
-            **kwargs))
+            await asyncio.sleep(0.05, loop=self.loop)
+        response = await asyncio.shield(
+            self._session.mgmt_request_async(
```

Comment:
do you need to pass the loop in here?

Reply:
pass `loop` into the `self._session.mgmt_request_async`? That method don't need a loop parameter to be passed in.
it would be using `self.loop` in the Session